backend/Zeus.Api.Application/Common/Interfaces/Authentication/IJwtGenerator.cs

```diff
@@ -0,0 +1,8 @@
+using Zeus.Api.Domain.UserAggregate;
+
+namespace Zeus.Api.Application.Common.Interfaces.Authentication;
+
+public interface IJwtGenerator
+{
+    string Generate(User user);
```

Comment:
Maybe you could make a difference between `GenerateAccessToken` and `GenerateRefreshToken` to be sure that a refresh can't be used an access and an access as a refresh ?